src/error_messages/debug_messages.h

```diff
@@ -12,82 +12,79 @@
-#define VU_HOTFIX_VUL         "(5533): Agent %s is vulnerable to %s. Condition: %s"
-#define VU_HOTFIX_INSTALLED   "(5534): Agent %s has installed %s that corrects the vulnerability %s."
-#define VU_UPDATE_JSON_FEED   "(5537): Updating from '%s'."
-#define VU_INDEX_TIME         "(5538): It took %ld seconds to %s vulnerabilities."
-#define VU_UPDATING_RH_YEAR   "(5539): Synchronizing the page %d from the vulnerability database."
-#define VU_INS_RH_SEC         "(5543): Inserting Red Hat section."
-#define VU_OSINFO_DISABLED    "(5544): Could not get the agent %s os_release, so its OS vulnerabilities will not be analysed. It may not have os_info scan activated."
-#define VU_ANAL_ACC_REPORTS   "(5545): Analyzing accumulated reports for %s in %s."
-#define VU_DISC_ACC_REPORTS   "(5546): Discarding report %s with check %s because %s is higher."
-#define VU_ADD_ACC_REPORTS    "(5547): Setting report %s with check %s because %s is lower."
-#define VU_NO_VER_REPORT      "(5548): There is no check version in %s, so the vulnerability will be reported directly."
-#define VU_OSINFOLNX_DISABLED "(5549): Could not get the agent %s os_version nor release, so its OS and kernel vulnerabilities will not be analysed. It may not have os_info scan activated."
-#define VU_UPDATE_PACK_NAME   "(5553): Inserting package names."
-#define VU_DOWNLOAD_START     "(5554): Trying to download '%s'."
-#define VU_DOWNLOAD_PAGE_SUC  "(5555): Page %d successfully downloaded."
-#define VU_NVD_UPD_CANCEL     "(5565): The %s update has failed, so the NVD feed will not be updated."
-#define VU_FIND_NVD_VULN_ERR  "(5566): It was not possible to look for the vulnerabilities of %s (%s) in the NVD."
-#define VU_INSERT_DATA_ERR    "(5567): It was not possible to insert %s in the agent software table."
-#define VU_OVAL_OBJ_INV       "(5569): Invalid OVAL object type: %s."
-#define VU_INS_VARIABLES      "(5570): Inserting variables..."
-#define VU_DISCARD_MATCH_N    "(5572): The matching configuration node '%s' from %s will be discarded because it is contemplated by '%s' (%s%s%s%s%s%s%s)."
-#define VU_AG_FULL_SCAN       "(5573): A full scan will be run on agent %s."
-#define VU_AG_PART_SCAN       "(5574): A partial scan will be run on agent %s."
-#define VU_NO_PACKAGE_SCAN    "(5576): The package inventory of the agent %s is not available, but a hotfix analysis will be launched."
-#define VU_NO_HOTFIX_AVAIL    "(5577): It is not possible to perform a hotfix scan on agent %s."
-#define VU_NO_HOTFIX_FEED     "(5578): We have not found a hotfix that solves %s for agent %s in the Microsoft feed, so it is not possible to know it is vulnerable."
-#define VU_PACKAGE_INSERT     "(5580): Agent (%s): Vulnerable package '%s' inserted into CVE '%s'"
-#define VU_DUPLICATED_PACKAGE "(5581): Agent (%s): Trying to insert duplicated vulnerable package '%s' into CVE '%s'"
+#define VU_STARTING_UPDATE    "(5400): Starting '%s' database update."
```

Comment:
Message IDs are duplicated between this file and `error_messages.h`. It would be better if they are unique.